app/views/shared/_terms_of_deposit.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="modal-content">
       <div class="row justify-content-between modal-header">
           <div class="col-5">
-            <h3 class="modal-title" id="terms-of-deposit-prompt">Terms of Deposit</h3>
+            <h1 class="modal-title" id="terms-of-deposit-prompt">Terms of Deposit</h1>
```

Comment:
I wonder if it's OK for the size of this text to bump up significantly, or if we want to use the `<h1>` element for semantics and use CSS to size the text closer to that of an `<h3>`. I believe adding the `h3` CSS class might do this (via bootstrap)?